packages/chains/src/bsc/plugins.ts

```diff
@@ -1,12 +1,12 @@
-import { assetSymbols, DeployedPlugins, underlying } from "@midas-capital/types";
+import { assetSymbols, DeployedPlugins, Strategy, underlying } from "@midas-capital/types";
 
 import assets from "./assets";
 
 const deployedPlugins: DeployedPlugins = {
   "0x10C90bfCFb3D2A7ae814dA1548ae3a7fC31C35A0": {
     market: "0x34ea4cbb464E6D120B081661464d4635Ca237FA7",
     name: "Bomb",
-    strategy: "BombERC4626",
+    strategy: Strategy.Bomb,
```

Comment:
Don't particularly love to add yet another property here that is quite similar to others, but I am OK with the solution for now

Reply:
It's the same property but more strictly typed, so it we can make sure if we add new plugins we use a valid value.